src/index.css

```
@@ -83,3 +97,56 @@ All colors MUST be HSL.
     font-family: Helvetica, Arial, 'Noto Sans Devanagari', sans-serif;
   }
 }
+
+@layer components {
+  .donate-heart {
+    position: absolute;
+    bottom: -0.125rem;
+    width: 0.75rem;
+    height: 0.75rem;
+    color: hsl(var(--primary-foreground) / 0.72);
+    fill: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)